person.rb

```diff
@@ -1,26 +1,33 @@
 require './corrector'
+require './classroom'
 
 class Person
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
- [ ] Same issue here, just as above. See suggestion below:

```suggestion
  attr_accessor :name, :age, :rentals
  attr_reader :id
```